platforms/minigame/common/engine/AssetManager.js

```diff
@@ -150,14 +150,13 @@ function loadFont (url, options, onComplete) {
 }
 
 function doNothing (content, options, onComplete) {
-    // exists(content, (existence) => {
-    //     if (existence) {
-    //         onComplete(null, content);
-    //     } else {
-    //         onComplete(new Error(`file ${content} does not exist!`));
-    //     }
-    // });
-    onComplete(null, content);
+    exists(content, (existence) => {
```

Comment:
Now this function has logic, is it possible to still call it `doNothing()`?

Reply:
Not sure. I didn't modify the code here. I guess call `doNothing()` is to unify the naming? @holycanvas

Reply:
Some asset needs post processing after downloading the file. and most of asset does not need post processing, So the callback function is named doNothing.

Reply:
Ok.